addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/TankerkoenigHandler.java

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.tankerkoenig.handler;
+
+import static org.openhab.binding.tankerkoenig.TankerkoenigBindingConstants.*;
+
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.tankerkoenig.TankerkoenigBindingConstants;
+import org.openhab.binding.tankerkoenig.internal.config.LittleStation;
+import org.openhab.binding.tankerkoenig.internal.config.OpeningTimes;
+import org.openhab.binding.tankerkoenig.internal.data.TankerkoenigDetailService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link TankerkoenigHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Dennis Dollinger/Jürgen Baginski
+ */
+public class TankerkoenigHandler extends BaseThingHandler {
+    private Logger logger = LoggerFactory.getLogger(TankerkoenigHandler.class);
+
+    private String apiKey;
+    private boolean setupMode;
+    private boolean use_OpeningTime;
```

Comment:
Please follow the naming guidelines, maybe search for occurrences of this word :-)